AccountService.Domain/Services/ServiceAccount.cs

```diff
@@ -42,15 +41,17 @@ public async Task RegisterUser(RegisterApiModel userRequest)
                 if (user.UserData.ServiceNumber == null)
```

Comment:
you can delete this check because only the participant can register

Reply:
alright, done